chatterbot/conversation/statement.py

```diff
@@ -19,6 +19,9 @@ def __init__(self, text, **kwargs):
         self.text = text
         self.in_response_to = kwargs.pop('in_response_to', [])
 
+        # Store tags information into statements
+        self.tags = kwargs.pop('tags', '')
```

Comment:
In both of these places, I'm fairly certain that the logic you are looking for would actually be something like:

```
for conversation in corpus:
    for text in conversation:
        statement = Statement(text=text)
        statement.add_tags(corpus.categories)
```